src/modules/AddOnProduct.rb

```diff
@@ -2209,6 +2224,128 @@ def add_rename(old_name, new_name)
 
   private
 
+    # Determine whether a rename for a product was added
+    #
+    # This method will return true if a rename was added through
+    # the #add_rename method (for example, a rename taken from
+    # SCC).
+    #
+    # @param old_name [String] Old product's name
+    # @param new_name [String] New product's name
+    # @return [Boolean] +true+ if the product was renamed; otherwise, +false+.
+    #
+    # @see #renamed_at?
+    def additionally_renamed?(old_name, new_name)
+      log.info "Search #{old_name} -> #{new_name} rename in added renames map: " \
+        "#{added_product_renames.inspect}"
+      renamed_at?(added_product_renames, old_name, new_name)
+    end
+
+    # Determine whether a product was renamed using libzypp
+    #
+    # libzypp (through #ResolvableProperties and #ResolvableDependencies method) is used
+    # to determine whether the product was renamed or not.
+    #
+    # @param old_name [String] Old product's name
+    # @param new_name [String] New product's name
+    # @return [Boolean] +true+ if the product was renamed; otherwise, +false+.
+    #
+    # @see #renamed_at?
+    def renamed_by_libzypp?(old_name, new_name)
+      renames = product_renames_from_libzypp
+      log.info "Search #{old_name} -> #{new_name} rename in libzypp: " \
+        "#{renames.inspect}"
+      renamed_at?(renames, old_name, new_name)
+    end
+
+    # Determine whether a product rename is included in the fallback map
+    #
+    # @see PRODUCT_RENAMES_FALLBACK_MAP
+    # @see #renamed_at?
+    def renamed_by_default?(old_name, new_name)
```

Comment:
NP: maybe `fallback_rename?` or something else. It's fallback (in the constant name) vs. default (in the method name), that's a bit inconsistent...